dll/win32/msgina/lang/es-ES.rc

```diff
@@ -1,5 +1,10 @@
-/* Spanish translation by Ismael Ferreras Morezuelas (2015-01-02) */
-LANGUAGE LANG_SPANISH, SUBLANG_NEUTRAL
```

Comment:
In case the CI error isn't clear: it's pointing out that you're now missing this line, therefore the compiler thinks these strings are English.